### DISCUSSION TITLE
IS FUXA CAN WRITE DATA TO MODBUS TCP ?

### DISCUSSION THREAD

**Discussion by NarutSH** [frangoteam/FUXA#1476] [Q&A]
I have already read the data from Modbus. 

However, I need to know if FUXA can write data to Modbus TCP, and **HOW ?**

**Comment by unocelli** [comment]
Hi, look the videos

**Reply by NarutSH** [reply]
Where? I think you forgot the link.

**Reply by unocelli** [reply]
The link are in the README -> Usage
